source/use-cases/home-resolver.rst

```diff
@@ -1,6 +1,128 @@
 Resolver for Home Networks
+==========================
+
+
+Setting up Unbound
+------------------
+
+Unbound is a powerful validating, recursive, caching DNS resolver. It’s used by some of the biggest tech companies in the world and also by home users who use it for Pi-hole or self-run resolvers. Setting it up for your home network can be quite simple as we’ll showcase below!
+
+Setting up your own DNS resolver for your entire home network requires a couple of things. Namely, a recursive DNS resolver (who knew!), and a dedicated machine where the resolver runs, which is always on and accessible to the entire network. This can be as simple as a Raspberry Pi connected to your home router or any other machine that you is always online!
+
+Because of the variaty of machines that Unbound can run on we cannot create a comprehensive tutorial for all possible options. For the extent of this tutorial we will use Ubuntu as a stepping stone which you could use for other machines. For this tutorial Ubuntu 20.04.1 LTS was used.
+
+While you could download the code and build it yourself if you wanted, it can be as simple as running:
+::
+	apt install unbound
+
+This gives you a full, compiled, and running version of Unbound which behaves as a caching recursive DNS resolver out of the box for the local machine.
+
+Do note that the current setup is only reachable on this machine.
+
+Testing the server locally
 --------------------------
 
-Using Unbound as the resolver for a home network.
+To verify that the server works correctly it’s a good idea to test it before committing the entire network to it. Luckily we can test this on the machine that you installed Unbound on (local) and from any other machine (remote)  that will be using the resolver after we expose Unbound to the network.
+
+The command for local testing is:
+::
+	dig example.com @127.0.0.1
+
+So we tell the dig tool to look up the IP address for example.com, and to ask this information to the server running at the ip address 127.0.0.1, which is where our Unbound machine is running.
+
+
+Setting up for a single machine
+-------------------------------
+
+Now that we have configured and tested our Unbound server, we can tell our machine to use it by default. We do this at:
+::
+	sudo nano /etc/systemd/resolved.conf
+In resolved.conf we change the entries under [Resolve] to use our own instance running at 127.0.0.1 
+::
+	sudo systemctl restart systemd-resolved
+
+We then need to stop the currently running pre-installed resolver:
+::
+	sudo systemctl disable systemd-resolved.service
+	sudo systemctl stop systemd-resolved
+
+And now we can then start using our new configuration:
+::
+	  sudo service network-manager restart
+
+And as a quick test a *dig* without specifying our Unbound server should give the same result as specifying it (with the *@127.0.0.1*)!
+::
+	dig example.com
```

Comment:
You can add the dig output, especially the `;;SERVER: ....` section to showcase that the local unbound was contacted by dig. Probably the before and after dig would show different IPs; at least this is what happens on my system (not Ubuntu 20.04) where the system resolver is listening on 127.0.1.1#53.